src/stop.sh

```diff
@@ -107,12 +107,28 @@ do_umountall() {
         [ -n "$CEPH_MNT" ] && sudo umount -f $CEPH_MNT
       fi
     done
+}
+
+do_umount_all_fuse_client() {
+    for fuse_pid in `pgrep ceph-fuse`; do
+        if test "$(sudo pwdx $fuse_pid | awk '{print $2}')" = `pwd`; then
```

Comment:
This is prone to bugs. User not always mounts and runs the stop.sh from the specific path. The comparison fails in that case.

Reply:
Isn't stop.sh written with assumption that it will be executed with `build` as its CWD? For example, `CEPH_BIN` is set to ` bin` [here](https://github.com/ceph/ceph/blob/main/src/stop.sh#L24), which won't work if CWD is not `build`.

Reply:
The `CEPH_BIN` is to run `stop.sh` script but the `mount` can be done from any directory and the CWD of the mount varies on that.

Reply:
Oh, okay. What you meant in `User will not always mount and run stop.sh from same path, so the comparison would fail`.

True, it would fail in such a case but that is beyond scope of this PR. Even beyond scope of stop.sh.

Even now when no MDS is not active on cluster, such mounts are left as it is. There's no way for `stop.sh` can safely dismount CephFS in such cases.

Reply:
Why not just warn the user that stale mounts have been left behind which have to be cleaned up manually.

Reply:
Okay, I can add code for that too. What about code this PR adds? Is it redundant or buggy in your view

Reply:
Well, ideally stop.sh should just warn users if stale mountpoints are left irrespective of MDSs being available (or not).